cmd/juju/secrets/list.go

```diff
@@ -115,6 +115,26 @@ type secretDisplayDetails struct {
 	Error            string                  `json:"error,omitempty" yaml:"error,omitempty"`
 	Value            *secretValueDetails     `json:"content,omitempty" yaml:"content,omitempty"`
 	Revisions        []secretRevisionDetails `json:"revisions,omitempty" yaml:"revisions,omitempty"`
+	Grants           []GrantInfo             `yaml:"grants,omitempty" json:"grants,omitempty"`
```

Comment:
We should use "access" instead of "grants". "access" is the standard terminology used elsewhere.